lock.js

```diff
@@ -0,0 +1,64 @@
+var Redlock = require('redlock');
```

Comment:
My only concern here is the code format

Reply:
I agree with @damsal7 on the code format, but bcx code is different from platform so we're probably not the right people to be asked. :D

Reply:
Just run the linter